main.py

```diff
@@ -145,15 +148,21 @@ def discover_delete_images(region_name, clusters):
 
         print("Number of running images found {}".format(len(running_sha)))
         ignore_tags_regex = re.compile(IGNORE_TAGS_REGEX)
+        ignore_repo_regex = re.compile(IGNORE_REPO_REGEX)
 
         for index, image in enumerate(tagged_images):
             if image['imagePushedAt'] < time_limit or index >= IMAGES_TO_KEEP:
                 for tag in image['imageTags']:
                     if "latest" not in tag and ignore_tags_regex.search(tag) is None:
+                        # Add a condition to exclude repo that contain the name regex
                         if not running_sha_set or image['imageDigest'] not in running_sha_set:
+                            image_url = repository['repositoryUri']
```

Comment:
Can we skip repo after line 109? as we here iterating over tags in repo that we will skip and we know it already at 109